Initial update (#1)
dime-worldbank/iesurveykit: README.md

# iesurveykit - Tools for high quality Impact Evaluation Surveys

> The **`iesurveykit`**  is a package of resources for primary data collection.

Reviewer: ```suggestion
The **`iesurveykit`**  is a package of tools to support high-quality primary data collection.
```

> This goes hand in hand with [iefieldkit](https://github.com/worldbank/iefieldkit).

Reviewer: ```suggestion
This goes hand in hand with [iefieldkit](https://github.com/worldbank/iefieldkit), a suite of Stata commands related to processing primary data. 
```

### Content

> iesurveykit currently supports three major components of the survey workflow survey design; survey completion; and data cleaning and survey harmonization.

Reviewer: ```suggestion
iesurveykit currently supports three major components of the survey workflow: (1) questionnaire design; (2) survey data quality monitoring; and (3) data cleaning and survey harmonization.
```

> * [Survey data quality checks templates (including real time data quality checks dashboard)](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Checks)

Reviewer: ```suggestion
* [Templates for survey data quality checks, including a dashboard for real-time data checks)](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Checks)
```

> iesurveykit currently supports three major components of the survey workflow: (1) questionnaire design; (2) survey data quality monitoring; and (3) data cleaning and survey harmonization.

Reviewer: what do we have on (3) for the time being? if nothing suggest not mentioning. the links that are included are out of order (survey checks comes before questionnaire design. might be clearer to put them as bullet points under each item in the numbered list.

Author: on 3 we have the iefieldkit command iecodebook which we can reference here. Included in readme

> iesurveykit currently supports three major components of the survey workflow: (1) questionnaire design; (2) survey data quality monitoring; and (3) data cleaning and survey harmonization.

Reviewer: ```suggestion
iesurveykit currently includes three major components

1. questionnaire design tools; 
**  [Guidelines on questionnaire design](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Form%20Design)
2.  iesurveychecks
* * A toolkit including template for a dashboard for real-time survey checks and scripts for data quality checks](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Checks)
3. data cleaning and survey harmonization.
```

Reviewer: suggest organizing it as above so the linkages are very clear. my formatting above is probably off as i was just writing in the comment field but hopefully the idea is clear. for consistency with other repos i think we want to organize this around the component tools and then describe them by how they fit in the workflow.

> * [Guidelines on Survey form design](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Form%20Design)

Reviewer: ```suggestion
* [Guidelines on questionnaire design](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Form%20Design)
```

Reviewer: or maybe, electronic survey instrument design?

> * [Templates for survey data quality checks, including a dashboard for real-time data checks)](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Checks)

Reviewer: ```suggestion
* [Templates for survey data quality checks, including a dashboard for real-time data checks](https://github.com/dime-worldbank/iesurveykit/tree/initial-update/Survey%20Checks)
```
